pkg/notes/notes.go

```diff
@@ -109,48 +109,48 @@ type ReleaseNotes map[int]*ReleaseNote
 // ReleaseNotesHistory is the sorted list of PRs in the commit history
 type ReleaseNotesHistory []int
 
-// GithubApiOption is a type which allows for the expression of API configuration
+// GitHubAPIOption is a type which allows for the expression of API configuration
 // via the "functional option" pattern.
 // For more information on this pattern, see the following blog post:
 // https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis
-type GithubApiOption func(*githubApiConfig)
+type GitHubAPIOption func(*gitHubAPIConfig)
```

Comment:
`gitHub` in `gitHubAPIConfig` feels awkward.
Was there a linter error that suggested it over `github` --> `githubAPIConfig`?

i.e., I'm aware of some guidance about maintaining proper names (prefer `GitHub` over `Github`), but it's kind of in violation anyway once it's an unexported variable.

(Same for the changes below.)

Reply:
Yep I agree, I'll change it to `githubAPIConfig`

Reply:
Thanks!